### PR TITLE
Fit transform adjustments

### PR DESCRIPTION
**Pull request by VeraZab**

### CODE REVIEW

File: src/EditorControls.js [hunk 1]
```diff
@@ -116,7 +116,9 @@ class EditorControls extends Component {
         if (this.props.beforeAddTrace) {
           this.props.beforeAddTrace(payload);
         }
-        graphDiv.data.push({type: 'scatter', mode: 'markers'});
```
Comment (nicolaskruchten) [inline src/EditorControls.js]: what's the reasoning for this change?

Comment (VeraZab) [reply src/EditorControls.js]: a fit is a new trace in the data array, so I need to be able to have the new trace be something other than 'scatter'

**Comment by nicolaskruchten** [comment]
💃

**Comment by VeraZab** [comment]
ok @nicolaskruchten this is ready for another review

File: src/styles/components/widgets/_numeric-input.scss [hunk 2]
```diff
@@ -18,7 +18,6 @@
 .numeric-input__wrapper {
   line-height: 20px;
   max-width: 100%;
-  width: 100%;
```
Comment (nicolaskruchten) [inline src/styles/components/widgets/_numeric-input.scss]: this causes some visual diffs in Percy that don't seem awesome...

**Comment by nicolaskruchten** [comment]
You'll have to walk me through some of these changes :)

File: src/components/containers/TraceAccordion.js [hunk 3]
```diff
@@ -12,11 +12,23 @@ const TraceFold = connectTraceToPlot(PlotlyFold);
 class TraceAccordion extends Component {
   render() {
     const {data = [], localize: _} = this.context;
-    const {canAdd, canGroup, children, messageIfEmptyFold} = this.props;
+    const {canAdd, canGroup, children, messageIfEmptyFold, panel} = this.props;
```
Comment (nicolaskruchten) [inline src/components/containers/TraceAccordion.js]: I would recommend changing `panel` to `excludeFits` or something as a boolean, just to match `canAdd` and `canGroup`

File: src/EditorControls.js (hunk 1 above)
Comment (nicolaskruchten) [reply src/EditorControls.js]: if you're actually not using this, let's revert it :)

**Pull request merged**